src/pages/GamePlay/Game.tsx

```
@@ -328,66 +304,143 @@ const GameList = ({
+    startIndex: index,
+    align: 'center',
+    axis: 'y',
+    containScroll: false,
+    skipSnaps: true,
+    watchSlides: false,
+  });
+
+  const cards = teams
+    .flatMap((t, index) => [
+      // Guild Rules Card
+      <FlipGuildCard guild={t.guild} />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)